runner/runner.go

```diff
@@ -144,7 +148,48 @@ func (r *Runner) Run(ctx context.Context, args []string) int {
 	// application & backend setup should be done before starting any new goroutines.
 	r.application.Setup()
 
-	var err error
+	// START - OpenTelemetry setup
+	otelEndpoint := config.GetString("OpenTelemetry.Endpoint", "") // @TODO check these env vars with infra
+	otelMetricsExportsInterval := config.GetDuration("OpenTelemetry.MetricsExportInterval", 10, time.Second)
+	if otelEndpoint == "" {
+		r.logger.Errorf("OpenTelemetry endpoint not provided")
+		return 1
```

Comment:
This is making some tests fail, we could skip the OpenTelemetry setup altogether if an endpoint is not provided. If provided an we can't connect then we fail. I'll make the changes but I'll leave the comment here for visibility.